src/abstract-field/abstract-field.component.ts

```diff
@@ -76,15 +70,12 @@ export abstract class AbstractFieldComponent
     return this.pathCache[key];
   }
 
-  get isErrorTooltipEnabled(): boolean {
-    return this.hasErrors;
-  }
-
   get pathString(): string {
     return this.pathUtilService.toPathString(this.path);
   }
 
-  get hasErrors() {
```

Comment:
this hould be removed but should be changed to

`return this.externalErrors.length > 0`